### PR TITLE
Vimeo A-Frame component

### PR DESCRIPTION
**Pull request by juniorxsound**
Creating an A-Frame component that wraps the [vimeo-threejs-player](https://github.com/vimeo/vimeo-threejs-player) and adding new documentation.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    volume: { type: 'number', default: 1.0 },
+    leftEye: { type: 'selector' },
+    rightEye: { type: 'selector' },
+    quality: { type: 'string', default: 'auto' }
+  },
+  init: function () {
+    console.log('Hello Vimeo 🎥 version: ', Vimeo.version)
+
+    if (this.data.leftEye && this.data.rightEye) {
+      console.log('[Vimeo] loading stereo spherical video 🌐')
```
Comment (caseypugh) [inline src/index.js]: Do we want to always console logging in the core component?

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,5 @@
+VIMEO_TOKEN=your_vimeo_token
+PORT=3333
+TESTS_PORT=4444
+DOMAIN=localhost:3333
```
[the remark this answers is not in the document]
Comment (juniorxsound) [reply .env.sample]: https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/2206ae76841899ffec363c84bf566c83bfd86122

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,5 @@
+VIMEO_TOKEN=your_vimeo_token
+PORT=3333
+TESTS_PORT=4444
```
[the remark this answers is not in the document]
Comment (juniorxsound) [reply .env.sample]: This environment variable is actually not needed in this repo and was just carried over from the [threejs-player repo](https://github.com/vimeo/vimeo-threejs-player) - removing this env var here and will change this to `TEST_PORT` in the threejs repo

[the remark this answers is not in the document]
Comment (juniorxsound) [reply .env.sample]: https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/fe6cedc9996e546784592136b77ed54fb043b1df

File: README.md [hunk 4]
```diff
@@ -1,73 +1,63 @@
...
+  <a-entity vimeo="id: vimeo_video_id">
+ ```
+Try our other examples or head to our getting started guide to learn more
 
-#### 4. Customize your world
-Never used [A-frame](https://aframe.io/) before? Head on over to [aframe.io](https://aframe.io/) to learn how to build your own WebVR experience!
+> Streaming Vimeo videos requires video file access via the Vimeo API. Accessing video files is limited to [Vimeo Pro and Business](https://vimeo.com/upgrade) customers.
 
+## Questions
+For questions and support, ask on [StackOverflow](https://stackoverflow.com/questions/ask/?tags=vimeo)
```
[the remark this answers is not in the document]
Comment (juniorxsound) [reply README.md]: https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/ccb2b15cc3ed87dae8fbd2cfadafeba2e83765a2

File: package.json [hunk 5]
```diff
@@ -1,20 +1,33 @@
...
+    "standard": "^12.0.1",
+    "sync-glitch-cli": "^2.0.1",
+    "webpack": "^4.12.2",
+    "webpack-cli": "^3.2.3",
+    "yargs": "^12.0.5"
   },
   "scripts": {
-    "build": "webpack",
-    "start": "webpack && node server.js"
+    "start": "node server.js",
```
[the remark this answers is not in the document]
Comment (juniorxsound) [reply package.json]: https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/b6aa67a592a00c98ce17722c7049b13ead4f0a95

File: assets/css/app.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  background: #fff;
+  color: #000;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#nav {
+	position: fixed;
```
[the remark this answers is not in the document]
Comment (juniorxsound) [reply assets/css/app.css]: https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/1714d13f0264ec058b884b9e4122aa5627fbfcad

File: src/index.js (hunk 1 above)
Comment (juniorxsound) [reply src/index.js]: https://github.com/vimeo/aframe-vimeo-component/pull/10/commits/2438354729bcea92840ba716e9fd89d4d45c7099

**Comment by juniorxsound** [comment]
@caseypugh Thanks for the review 🙌

About warning and pointing to the Getting Started, I think it's a great idea, I added a [log when the host is not our Glitch example repo here](https://github.com/vimeo/aframe-vimeo-component/blob/vimeo-aframe-component/assets/js/host-console-warning.js). I also thought since our left nav is full height, and currently has only 3 examples, I think it be valuable to also have a _Guides_ list under the _Examples_ list in there.

What do you think about that?

**Pull request merged**